config/environment.js

```diff
@@ -1,7 +1,7 @@
 /* jshint node: true */
 
 module.exports = function(environment) {
-  var ENV = {
+  const ENV = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).